tests/unit/test_computations.py

```diff
@@ -184,3 +184,105 @@ def test_moving_median(window_size: int, expected_output: np.ndarray) -> None:
     assert np.all(
         np.isclose(test_result.measurements, expected_output)
     ), "Test results do not match the expected output"
+
+
+def test_resample_downsample_simple() -> None:
+    """Test the downsampling happy-path."""
+    time = [datetime(1990, 1, 1) + timedelta(seconds=secs) for secs in range(4)]
+    measurement = models.Measurement(
+        measurements=np.array([1, 2, 3, 4]),
+        time=pl.Series(time),
+    )
+    delta_t = 2
+    expected = models.Measurement(
+        measurements=np.array([1.5, 3.5]), time=pl.Series("time", time[0::2])
+    )
+
+    actual = computations.resample(measurement, delta_t)
+
+    assert np.allclose(actual.measurements, expected.measurements)
+    assert all(actual.time == expected.time)
+
+
+def test_resample_downsample_mismatch_time_start() -> None:
+    """Test the downsampling with 60 and 30Hz.
+
+    This test is specifically to catch any issues that arise with sampling
+    associated with imprecise detla_t values.
+    This test will fail without the `start_by` option in the
+    polars`group_by_dynamic` function used by computations.resample.
+    """
+    time = [
+        datetime(1990, 1, 1) + timedelta(microseconds=16666 * secs)
+        for secs in range(100)
```

Comment:
Some confusing variable naming here, since you're multiplying it by 16666 microseconds, its really more of a counter than a 'second'. You could use a more generic name here `idx`, `index`, etc..